ui/src/app/workflows/components/workflow-details/artifact-panel.tsx

```diff
@@ -63,12 +70,16 @@ export const ArtifactPanel = ({
                 }>
                 <ErrorBoundary>
                     <div className='white-box'>
-                        <h3>{artifact.name}</h3>
+                        <h3>
+                            <i className='fa fa-file-alt' /> {artifact.name}
+                        </h3>
                         <p>
                             <small>{urn}</small>
                         </p>
                         {error && <ErrorNotice error={error} />}
-                        {show ? (
+                        {artifact.deleted ? (
+                            <p>Artifact has been deleted.</p>
+                        ) : show ? (
                             <ViewBox>
```

Comment:
This is interesting. It's actually doing the work that is from this issue, which I worked on and have a PR for: https://github.com/argoproj/argo-workflows/issues/8666

Alex's point of view was that we shouldn't just go by the 'deleted' flag but instead we should go to S3 to determine whether it's actually been deleted or not. So, in that PR basically if the backend returns a 404 (FileNotFound) error, then it shows "File not found", and he thought that was sufficient, not requiring UI changes.

Still, it seems reasonable to me that if GC deleted the Artifact and set the deleted flag, then the only thing that would cause the Artifact to re-exist would be a new run of that Workflow, and in that case it seems reasonable to call the original Artifact from the original Workflow 'deleted'. So, maybe this is fine.